src/summary/history/encounter.resource.tsx

```diff
@@ -0,0 +1,235 @@
+// import { openmrsFetch } from "@openmrs/esm-api";
+
+export function encounterResource(
+  patientIdentifer: string,
+  abortController: AbortController
+) {
+  //   return openmrsFetch(
+  //     `/ws/fhir/Encounter?patient.identifier=${patientIdentifer}`,
+  //     { signal: abortController.signal }
+  //   );
+  return Promise.resolve(mockPatientEncounterSearchResponse);
+}
+const mockPatientEncounterSearchResponse = {

```

Comment:
perhaps you can move this to the __mock__ tests folder and can reuse it later for tests as well?